cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/cats/sql/cache/SqlCache.kt

```diff
@@ -1,7 +1,6 @@
 package com.netflix.spinnaker.cats.sql.cache
```

Comment:
Everything in this file is a result of moving some internal naming convention code into a new `SqlNames` class.